<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OctoManagerView</class>
 <widget class="QDialog" name="OctoManagerView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OctopodProd Manager</string>
  </property>
  <widget class="QPushButton" name="btnAddClient">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbProjects">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CLIENTS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddProject">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSettings">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEditClient">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEditProject">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PROJETS</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbClients">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/OctoManagerBanner.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
